site/documentation/content/api/kafka-api/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Kafka-based APIs"
+weight: 416
+---
+
+The Kafka-based APIs of Eclipse Hono&trade; provide an alternative to the older APIs based on AMQP 1.0.
+With these APIs, clients publish data to as well as consume data from an Apache Kafka&reg; cluster instead of using an AMQP messaging network.
+
+<!--more-->
+
+{{% note title="Tech preview" %}}
+The support of Kafka as a messaging system is currently a preview and not yet ready for production. The APIs may change with the next version. 
+{{% /note %}}
+
+
+## Kafka-based Messaging
+
+Using Kafka instead of AMQP comes with slightly different behavior. Kafka provides a Publish/Subscribe messaging style.
+Every message is sent by a *producer* to a *topic* in a Kafka cluster, where it will be persisted by (multiple) *brokers*.
```

Comment:
in another issue we had been discussing kafka terminology. It seems that within the kafka ecosystem poducers are usually said to *write* to a topic. IMHO we should try to use the same consistent terminology across the documentation. WDYT?

Reply:
That is a tough question because from what I see the terminology used in the world of Kafka seems to be pretty inconsistent here. The terms "publish", "write", "produce", and "send" are often used quite interchangeably, including in Kafka's documentation. "Publish" and "write" are even marked as synonyms, the `Producer` methods are named "send"... The picture is similar with "messages" vs. "events" vs. "records". A brief research did not give me a clear picture. In any case, I wouldn't invest too much effort here.